mappings/com/mojang/realmsclient/client/Errable.mapping

```diff
@@ -1,3 +1,3 @@
-CLASS net/minecraft/class_5221
+CLASS net/minecraft/class_5221 com/mojang/realmsclient/client/Errable
```

Comment:
Are you sure that when the error methods are called, the object itself errors with the given message than listen to error from other things?

Reply:
Yes, that's exactly what I was thinking about that name. In one of my enigma branches I have a `Validatable` class because I added a validation framework, which does something very similar as far this as I can see, but it has the slight difference in meaning that it doesn't create the errors itself. That might actually be a good name for this interface. Or `ErrorReceiver` as in my PR but I still think that name is also kinda meh, definitely better than `Errable` though, imo.

Reply:
In fact `Errable` is the right name, as this object has the error when the `error` method is called. Imo `error` should be like `errored` instead